app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -58,6 +59,14 @@ def selfie_requirement_met?
         stored_result.selfie_check_performed?
     end
 
+    def resolve_doc_auth_vendor(user)
+      if user.establishing_in_person_enrollment || user.pending_in_person_enrollment
+        Idp::Constants::Vendors::USPS
```

Comment:
I think there are some edge cases that this will get mixed up on:

1. User starts IPP, backs out, and goes down the remote doc auth path
2. User starts IPP, abandons, then starts a new verification, selecting remote this time

In this case, we know the user has _not_ chosen IPP for doc auth, so I don't think we should write it here. We likely want to set this to 'usps' somewhere else later on--perhaps on submission of the "Verify your info" screen?

Reply:
That's a good point and I hadn't thought about that.

Reply:
Instead of doing this, converted to updating in `Idv::InPerson::StateIdController#update`. (Figuring this out was a bit of pain since it's a "hidden" step.)